code/CapstoneBackend/Model/Response.cs

```diff
@@ -9,7 +9,7 @@ public class Response<T>
         /// <summary>
         ///     The status code of the response
         /// </summary>
-        public int StatusCode { get; set; } = 200;
+        public uint StatusCode { get; set; } = 200;
```

Comment:
Its definitely safer, but it may not be necessary to ensure all positive integers are unsigned.
We could probably decide a style guide for our naming and strict types like this